test.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+
+
+
+int main (){
```

Comment:
write int `int main void
[git-cheat-sheet-education.pdf](https://github.com/junaid-saleem/repo-exercise/files/13693150/git-cheat-sheet-education.pdf)
@junimp look  
`

Reply:
- plz check this